samples/generate-offline-map-with-custom-parameters/src/main/java/com/esri/arcgismaps/sample/generateofflinemapwithcustomparameters/screens/GenerateOfflineMapWithCustomParametersScreen.kt

```diff
@@ -0,0 +1,308 @@
+                        modifier = Modifier.align(Alignment.CenterHorizontally)
+                    ) {
+                        Text("Reset to online map")
+                    }
+                }
+            }
+        },
+        // Floating action button to show the parameter overrides bottom sheet
+        floatingActionButton = {
+            if (!showBottomSheet && !mapViewModel.showResetButton) {
+                FloatingActionButton(
+                    modifier = Modifier.padding(bottom = 36.dp, end = 12.dp),
+                    onClick = { showBottomSheet = true }
+                ) {
+                    Icon(
+                        imageVector = Icons.Filled.Settings,
+                        contentDescription = "Parameter overrides"
+                    )
+                }
+            }
+        })
+}
+
+@Composable
+fun OverrideParameters(
+    defineParameters: (Int, Int, Int, Boolean, Boolean, Int, Boolean) -> Unit,
+    setBottomSheetVisibility: (Boolean) -> Unit
+) {
+    // Collection of parameter overrides to set in this composable
+    var minScale by remember { mutableFloatStateOf(15f) }
+    var maxScale by remember { mutableFloatStateOf(20f) }
+    var extentBufferDistance by remember { mutableFloatStateOf(150f) }
+    var includeSystemValves by remember { mutableStateOf(false) }
+    var includeServiceConnections by remember { mutableStateOf(false) }
+    var minHydrantFlowRate by remember { mutableFloatStateOf(500f) }
+    var cropToWaterPipeExtent by remember { mutableStateOf(false) }
+
+    Column(
+        modifier = Modifier
+            .wrapContentSize()
+            .background(MaterialTheme.colorScheme.background)
+            .padding(12.dp)
+            .verticalScroll(rememberScrollState()),
+        horizontalAlignment = Alignment.Start,
+    ) {
+        Text(
+            text = "Override parameters",
+            style = MaterialTheme.typography.titleMedium,
+            modifier = Modifier.align(Alignment.CenterHorizontally)
+        )
+
+        Text(text = "Adjust basemap", style = MaterialTheme.typography.labelLarge)
+        Row(
+                modifier = Modifier.fillMaxWidth(),
+                horizontalArrangement = Arrangement.spacedBy(4.dp),
+                verticalAlignment = CenterVertically
+            ) {
+                Text(text = "Min scale level:", style = MaterialTheme.typography.labelMedium)
+                Slider(
+                    value = minScale,
+                    // Don't let the min scale exceed the max scale
+                    onValueChange = {
+                        minScale = it
+                        if (minScale >= maxScale) {
+                            maxScale = minScale + 1
+                        }
+                    },
+                    valueRange = 0f..22f, modifier = Modifier.weight(1f),
+                    steps = 21
+                )
+                Text(text = "${minScale.toInt()}")
```

Comment:
slider seems to grow once the numbers  is > 9, i would suggest setting a fixed size